lib/Interpreter/InterpreterCallbacks.cpp

```diff
@@ -239,6 +240,17 @@ namespace cling {
     return false;
   }
 
+  void InterpreterCallbacks::CreateCodeCompleteConsumer(Interpreter* child) {
```

Comment:
ROOT now needs to
- create a CCC
- set it in the interpreter
- set it in Sema.

I'd prefer if ROOT only needs to do the first step, which is why I was arguing for returning a `unique_ptr` (or, if that's easier on the clang interfaces, just a CCC*) and doing the setting  on the caller side, i.e. in `ClingTabCompletion::Complete()`.